core/src/test/java/io/parsingdata/metal/EqualityTest.java

```diff
@@ -136,4 +137,17 @@ public void multiConstructorTypes() {
         }
     }
 
+    @Test
+    public void immutableList() {
+        final ImmutableList<String> object = ImmutableList.create("a");
```

Comment:
I use the terms object, same and other in the other test code around equality, so here I'm striving for consistency, but in another dimension :)